dataloader.py

```diff
@@ -109,8 +116,8 @@ def _resize_data(self, image, mask):
         """
         Resizes images to specified size.
         """
-        image = tf.image.resize(image, [self.image_size, self.image_size])
-        mask = tf.image.resize(mask, [self.image_size, self.image_size], method='nearest')
+        image = tf.image.resize(image, self.image_size)
+        mask = tf.image.resize(mask, self.image_size)
```

Comment:
TF requires image size to be a tuple (class accepts integer), so this needs to be a tuple, or the docstring needs to change

Reply:
when you call the dataloader class you will pass the size as tuple for example:
dataloader = Dataloader("/usr/local/images", "/usr/local/masks", (256,256))
that is fine :)

Reply:
BTW, default input argument shouldn't be mutable hence I changed the channels argument from `[3,3]` to `(3,3)`.  
Please check the new commit you can test it out but don't forget to pass image size as a tuple I also documented it so it should "scream" at you if you won't do it.